app/javascript/onboarding/utilities.js

```diff
@@ -6,3 +6,16 @@ export const jsonToForm = data => {
 
 export const getContentOfToken = token =>
```

Comment:
I don't believe we use this anywhere else but for the CSRF token, so consider just naming this `getCsrfToken` and omit the `token` parameter and hard-code `csrf-token` in the function. If in the future we need to get the content of different meta tags, then we should consider promoting this to a more generic function like you currently have.

Reply:
Also, consider adding some unit tests to these functions as well.